database/config.py

```diff
@@ -1,8 +1,2 @@
 ### config.py ###
 
-# For now, to test the database locally, you will have to set the URI below to log into your own database server of choice.
-# TODO Uncomment the below with your own server's information.
-# Scheme: "postgres+psycopg2://<USERNAME>:<PASSWORD>@<IP_ADDRESS>:<PORT>/<DATABASE_NAME>"
-
-# DATABASE_URI = "postgres+psycopg2://<USERNAME>:<PASSWORD>@<IP_ADDRESS>:<PORT>/<DATABASE_NAME>"
-DATABASE_URI = "postgres+psycopg2://postgres:bentopostgres@localhost:5432/megaqc1"
```

Comment:
DATABASE_URI = "postgres+psycopg2://USERNAME:PASSWORD@IP_ADDRESS:0/DATABASE_NAME"

Let's leave a fake DATABASE_URI in to prevent those missing DATABASE_URI errors. I know the connect command needs to be ran first - but right now those errors are messing up the falcon_multiqc --help.

It would be even better if we had this fake db string in a constants.py file or similar, then in the other commands we can check if the database has been configured properly or not before running. We can do this in another PR if you want (when we deal with making the tool less error prone).